src/App.vue

```diff
@@ -1,7 +1,13 @@
 <template>
-  <div id="app"></div>
+  <div id="app">
+    App Home Page
+  </div>
 </template>
 
+<script>
+export default {};
+</script>
+
```

Comment:
Please check and remove if we don't need this code

Reply:
That tag will be used soon when UI components added.

Reply:
okay a @The-Luc